packages/react-components/react-persona/MIGRATION.md

```diff
@@ -13,6 +13,7 @@ Here's how the API of v8's `Persona` compares to the one from v9's `Persona` com
 - ```
 
   ```
+
```

Comment:
Could you revert this change? you can use `--no-verify` when commiting so it doesn't change it back.